backend/src/core/models/steam.apps.aggregate.spec.js

```diff
@@ -66,27 +48,25 @@ describe("SteamAppsAggregate", function () {
       });
 
       it("the first app is correctly identified", function () {
-        expect(this.steamAppsArray.apps[0].appid).toBe(1);
-        expect(this.steamAppsArray.apps[0].triedVia).toEqual(["steamWeb"]);
-        expect(this.steamAppsArray.apps[0].failedVia).toEqual(["steamWeb"]);
-        expect(this.steamAppsArray.apps[0].type).toBe("unknown");
+        expect(this.steamAppsArray.content[0].appid).toBe(1);
```

Comment:
Tried using toEqual, but the object I compare it to has to be an instance of same class, seems like it would be unnecessarily extra work for no real benefit? Let me know if I should do it anyway, and I will do in the future.